lasagne/layers/conv.py

```diff
@@ -281,7 +281,7 @@ def __init__(self, incoming, num_filters, filter_size, stride=1, pad=0,
         self.num_filters = num_filters
         self.filter_size = as_tuple(filter_size, n, int)
         self.flip_filters = flip_filters
-        self.stride = as_tuple(stride, n, int)
+        self.stride = as_tuple(stride, n)
```

Comment:
Yeah...it's admittedly a not elegant solution to a python3 problem. Fact is, I don't have python3 installed and I went for a lazy solution. :)

For some reason in python3 `numpy.int32` and `numpy.int64` fail to be recognized as `int` (`isinstance(var, int)` returns `False`). This triggers [this exception](https://github.com/Lasagne/Lasagne/blob/996bf64c0aec6d481044495800b461cc62040041/lasagne/utils.py#L194-L196).

It will probably work if I replace the `int` with `(int, np.int32, np.int64)`. I can try